pkg/chunk/testutils/testutils.go

```diff
@@ -22,7 +23,7 @@ const (
 // Fixture type for per-backend testing.
 type Fixture interface {
 	Name() string
-	Clients() (chunk.IndexClient, chunk.ObjectClient, chunk.TableClient, chunk.SchemaConfig, error)
+	Clients() (chunk.IndexClient, chunk.Client, chunk.ObjectClient, chunk.TableClient, chunk.SchemaConfig, error)
```

Comment:
Having both `chunk.Client` and `chunk.ObjectClient` sounds redundant to me. Could you explain why we need `chunk.ObjectClient` too?

Reply:
I can have the downstream fixtures wrap the object client before returning them